sktime/forecasting/hf_patchtst_forecaster.py

```diff
@@ -0,0 +1,659 @@
+    ...     ).drop(columns = ["date"]
+    ... )
+    >>> from sklearn.preprocessing import StandardScaler
+    >>> scaler = StandardScaler()
+    >>> scaler.set_output(transform="pandas") # doctest: +SKIP
+    >>> scaler = scaler.fit(dataset_path.values) # doctest: +SKIP
+    >>> df = scaler.transform(dataset_path) # doctest: +SKIP
+    >>> df.columns = dataset_path.columns
+    >>> forecaster = HFPatchTSTForecaster(
+    ...     model_path="namctin/patchtst_etth1_forecast",
+    ...     config = {
+    ...         "patch_length": 8,
+    ...         "context_length": 512,
+    ...         "patch_stride": 8,
+    ...         "d_model": 128,
+    ...         "num_attention_heads": 2,
+    ...         "ffn_dim": 512,
+    ...         "head_dropout": 0.3,
+    ...         "prediction_length": 64
+    ...     },
+    ...     fit_strategy = "minimal",
+    ...     training_args = {
+    ...         "output_dir":"/PatchTST/",
+    ...         "overwrite_output_dir":True,
+    ...         "learning_rate":1e-4,
+    ...         "num_train_epochs":1,
+    ...         "per_device_train_batch_size":16,
+    ...     }
+    ... ) # doctest: +SKIP
+    >>> forecaster.fit(y = df, fh = list(range(1,63))) # doctest: +SKIP
+    >>> y_pred = forecaster.predict() # doctest: +SKIP
+
+    >>> #Example with a pre-trained model to do zero-shot forecasting
+    >>> from sktime.forecasting.hf_patchtst_forecaster import HFPatchTSTForecaster
+    >>> import pandas as pd
+    >>> dataset_path = pd.read_csv(
+    ...     "https://raw.githubusercontent.com/zhouhaoyi/ETDataset/main/ETT-small/ETTh1.csv"
+    ...     ).drop(columns = ["date"]
+    ... )
+    >>> from sklearn.preprocessing import StandardScaler
+    >>> scaler = StandardScaler()
+    >>> scaler.set_output(transform="pandas") # doctest: +SKIP
+    >>> scaler = scaler.fit(dataset_path.values) # doctest: +SKIP
+    >>> df = scaler.transform(dataset_path) # doctest: +SKIP
+    >>> df.columns = dataset_path.columns
+    >>> forecaster = HFPatchTSTForecaster(
+    ...     model_path="namctin/patchtst_etth1_forecast",
+    ...     fit_strategy = "zero-shot",
+    ...     training_args = {
+    ...         "output_dir":"/PatchTST/",
+    ...         "overwrite_output_dir":True,
+    ...         "learning_rate":1e-4,
+    ...         "num_train_epochs":1,
+    ...         "per_device_train_batch_size":16,
+    ...     }
+    ... ) # doctest: +SKIP
+    >>> forecaster.fit(y = df, fh = [1,2,3,4,5]) # doctest: +SKIP
+    >>> y_pred = forecaster.predict() # doctest: +SKIP
+    """
+
+    _tags = {
+        "X_inner_mtype": [
+            "pd.DataFrame",
+            "pd-multiindex",
+            "pd_multiindex_hier",
+        ],
+        "y_inner_mtype": [
+            "pd.DataFrame",
+            "pd-multiindex",
+            "pd_multiindex_hier",
+        ],
+        "scitype:y": "both",
+        "ignores-exogeneous-X": True,
+        "requires-fh-in-fit": True,
```

Comment:
```suggestion
        "requires-fh-in-fit": False,
```
model can work without `fh` in `fit`